analyzers/python/python.go

```diff
@@ -154,6 +155,15 @@ func (a *Analyzer) Analyze() (graph.Deps, error) {
 			Direct:     imports,
 			Transitive: fromImports(imports),
 		}, nil
+	case "pipenv":
+		imports, err := pipenv.FromFile("Pipfile.Lock")
+		if err != nil {
+			return graph.Deps{}, err
+		}
+		return graph.Deps{
+			Direct:     imports,
+			Transitive: fromImports(imports),
```

Comment:
This can't be right. We use `fromImports` for flat dependency lists, but `pipenv` has a meaningful notion of transitive dependencies.

See `pipenv graph` (in fact, you could probably parse this output as a strategy -- it looks it there is `pipenv graph --json` and `pipenv graph --json-tree`).

Reply:
I will take a look more at how pipenv manages dependencies, however, I had no problem extracting the correct information using `fromImports`. I could be wrong about this but I will let you know what I find out.

Reply:
Pipfile.lock is a flat dependency list and so thats why I chose this approach to begin with. You're absolutely right though, I wasn't prioritizing transitive dependencies and taking advantage of how fossa core displays them. `pipenv graph` looks like a better solutuion and will give us better information on dependencies.

The only downside with `pipenv graph` is that we can't get dev dependencies. I wasn't intending on implementing in the first place, but I do see a how this could be useful at some point in the future.